.github/workflows/republish.yml

```diff
@@ -0,0 +1,56 @@
+name: Publish sequencer.publiclab.org demo
+
+on:
+  release:
+    types: [created]
+jobs:
+  merge:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Merge upstream
+        run: |
+          git config --global user.name 'jywarren'
+          git config --global user.email 'jeff@unterbahn.com'
+          git pull --unshallow
+          git checkout gh-pages
+          git remote add upstream https://github.com/publiclab/image-sequencer.git
+          git fetch upstream
+          git reset --hard upstream/main
+          npm run setup --force
+
+          grunt production
+
+          if [ ! -f CNAME ];
+          then
+            touch CNAME
+          fi
+          echo "sequencer.publiclab.org" > CNAME
+
+          rm -R docs/
+          rm -R test/
+          rm CONTRIBUTING.md
+          rm index.js
+
+          cp src/ui/prepareDynamic.js prepareDynamic.js
+
+          rm -R src/
+          mkdir -p src/ui/
+          mv prepareDynamic.js src/ui/prepareDynamic.js
+
+          git add .
+          git add -f node_modules/jquery/
```

Comment:
Are these dependencies up-to-date?

Reply:
We cant add all, node modules are massive and there is a limit on Githubw. Should we just stop using npm for this and copy-paste the dist?

Reply:
Hmm. but copy-paste which? Could we... scan the .html files within `examples/` with `grep` for any references to `/node_modules/*` files? it'd be a bit brittle, i think... 

We could just add a checklist item or a text reminder to check for any new demo dependencies, i guess...

Reply:
By copy-paste, I mean that we can just get the required dist files (eg: jquery) and put them in the examples/ instead of using node_modules.

Alternatively, maybe we can use grunt to create one single bundle inside examples/.